static/js/tpos.js

```diff
@@ -1007,6 +1008,17 @@ window.app = Vue.createApp({
         .onCancel(() => {
           this.invoiceDialog.internalMemo = ''
         })
+    },
+    headerToggle() {
+      if (this.headerElement) {
+        this.headerElement.style.display =
```

Comment:
don't add inline styles like that, either not render the element, or add/remove classes

Reply:
q-header is extended from base.html -> public.html in tpos.html, no way to really hide/show on tpos